engine/src/main/scala/cromwell/webservice/PartialWorkflowSources.scala

```diff
@@ -35,7 +35,8 @@ final case class PartialWorkflowSources(workflowSource: Option[WorkflowSource] =
                                         customLabels: Option[WorkflowJson] = None,
                                         zippedImports: Option[Array[Byte]] = None,
                                         warnings: Seq[String] = List.empty,
-                                        workflowOnHold: Boolean)
+                                        workflowOnHold: Boolean,
+                                        requestedWorkflowIds: Option[Vector[WorkflowId]])
```

Comment:
We're counting on the `workflowInputs` and `requestedWorkflowIds` vectors having the same order? Looks like those are the only two things that vary for a multi-workflow batch.

Reply:
Yes, I'm asserting that the order of the IDs will match workflows representing the corresponding inputs.

@salonishah11 - are you saying that instead of an `Option` we can use an empty vector to indicate unsupplied? I think that would work, I can try it out...